actix-http/src/client/connector.rs

```diff
@@ -163,12 +163,41 @@ where
         self
     }
 
+    /// Maximum supported http major version
+    /// When supplied 1 both HTTP/1.0 and HTTP/1.1 will be allowed
+    pub fn max_http_version(mut self, val: u8) -> Self {
```

Comment:
I think this should take a `http::Version`.

Reply:
@robjtede indeed that's better. what about providing minimum cap at http/1.1 not growing the list below?
```
            http::Version::HTTP_09 => vec![b"http/0.9".to_vec()],
            http::Version::HTTP_10 => vec![b"http/1.0".to_vec()],
            http::Version::HTTP_11 => vec![b"http/1.1".to_vec()],
            http::Version::HTTP_2 => vec![b"h2".to_vec(), b"http/1.1".to_vec()],
```

Reply:
yeah sure keeping 1.1 as minimum sounds good

Reply:
Should be fixed now